### PR TITLE
Improved aetos logging 

### PR DESCRIPTION
**Pull request by Madanagopal19**
Increasing timeout for validating deployments

Branch build test pipeline:
https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds-branch-build-test/83/console

### CODE REVIEW

File: drivers/pds/lib/utils.go [hunk 1]
```diff
@@ -299,10 +312,10 @@ func CheckNamespace(namespace string) (bool, error) {
 					Labels: map[string]string{pxLabel: "true"},
 				},
 			}
-			log.Infof("Creating namespace %v", namespace)
+			log.InfoD("Creating namespace %v", namespace)
 			ns, err = k8sCore.CreateNamespace(nsName)
 			if err != nil {
-				log.Errorf("Error while creating namespace %v", err)
+				//log.FailOnError(err, "Error while creating namespace")
```
Comment (lsrinivas-pure) [inline drivers/pds/lib/utils.go:318]: remove the commented line if not required

File: drivers/pds/lib/utils.go [hunk 2]
```diff
@@ -333,36 +347,35 @@ func ReadParams(filename string) (*Parameter, error) {
...
-
-			log.Errorf("File error: %v\n", err)
 			return nil, err
+			//log.FailOnError(err, "File error: ")
 		}
 		err = json.Unmarshal(file, &jsonPara)
 		if err != nil {
-			log.Errorf("Error while unmarshalling json: %v\n", err)
 			return nil, err
+			//log.FailOnError(err, "Error while unmarshalling json:")
```
Comment (lsrinivas-pure) [inline drivers/pds/lib/utils.go:363]: same here

File: tests/pds/dataservice_test.go [hunk 3]
```diff
@@ -35,17 +35,15 @@ var _ = Describe("{DeletePDSPods}", func() {
...
-				Expect(err).NotTo(HaveOccurred())
-
-				log.Infof("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
+				log.FailOnError(err, "Error while getting resource template")
+				log.InfoD("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
 
 				dataServiceDefaultAppConfigID, err = pdslib.GetAppConfTemplate(tenantID, ds.Name)
-				Expect(err).NotTo(HaveOccurred())
+				log.FailOnError(err, "Error while getting app configuration template")
 				Expect(dataServiceDefaultAppConfigID).NotTo(BeEmpty())
```
Comment (lsrinivas-pure) [inline tests/pds/dataservice_test.go:43]: use dash verify here

**Review by bhsrinivasan-px — APPROVED** [review]
LGTM

File: tests/pds/dataservice_test.go [hunk 4]
```diff
@@ -961,22 +786,21 @@ var _ = Describe("{RestartPXPods}", func() {
...
 					Expect(deploymentPods).NotTo(BeEmpty())
 				})
 
 				var nodeList []*corev1.Node
 				Step("Get the node that the PV of the pod resides on", func() {
 					for _, pod := range deploymentPods {
-						log.Infof("The pod spec node name: %v", pod.Spec.NodeName)
 						log.InfoD("The pod spec node name: %v", pod.Spec.NodeName)
 						nodeObject, err := pdslib.GetK8sNodeObjectUsingPodName(pod.Spec.NodeName)
 						Expect(err).NotTo(HaveOccurred())
```
Comment (lsrinivas-pure) [inline tests/pds/dataservice_test.go:806]: use dash verifications here

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]
There are few places where expect is used, please have a look and modify them

File: drivers/pds/lib/utils.go (hunk 1 above)
Comment (Madanagopal19) [reply drivers/pds/lib/utils.go]: done

File: drivers/pds/lib/utils.go (hunk 2 above)
Comment (Madanagopal19) [reply drivers/pds/lib/utils.go]: done

File: tests/pds/dataservice_test.go (hunk 3 above)
Comment (Madanagopal19) [reply tests/pds/dataservice_test.go:43]: Im checking this dataServiceDefaultAppConfigID not to be empty, not sure with dash we can verify it.

File: tests/pds/dataservice_test.go (hunk 4 above)
Comment (Madanagopal19) [reply tests/pds/dataservice_test.go]: done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**